plugins/gatsby-theme-iterative-docs/src/utils/front/safeLocalStorage.ts

```diff
@@ -0,0 +1,20 @@
+const IS_SUPPORTED = ((): boolean => {
+  try {
+    const key = 'localstoragecheck'
+    localStorage.setItem(key, key)
+    localStorage.removeItem(key)
+    return true
+  } catch (e) {
+    return false
+  }
+})()
```

Comment:
maybe simply do this?
```js
return typeof(Storage) !== "undefined"
```

Reply:
I think the issue isn't about the fact that `Storage` doesn't exist. It's about how attempting to use `Storage` will cause an error. 🤔 Which is why the function check actually attempts to use it.